fastai/imagito/utils.py

```diff
@@ -0,0 +1,30 @@
+from typing import Iterable
+
+import funcy
+import numpy as np
+import os
+import pandas as pd
+import re
+import time
+import pickle
+import gzip
+
+
+def get_date_str(seconds=False):
+    if seconds:
+        return time.strftime('%Y-%m-%d-%H%M:%S')
```

Comment:
I feel slightly better always having seconds just to guarantee we never have collisions